e2e/tslint.json

```diff
@@ -46,6 +46,9 @@
     "no-consecutive-blank-lines": [
       true,
       2
+    ],
+    "triple-equals": [
```

Comment:
Why are you disabling this instead of making the code conform with this rule?

Reply:
Because tslint fails without this option for this code https://github.com/all-of-us/workbench/blob/959d28198cd80fb3ac0c451b35bb6f4b2574fb51/e2e/tests/notebook/notebook-download.spec.ts#L18
From CH: the `==` is intentional, treat `null` as false.

Reply:
Right, but why not do a null check instead? It's worth it to not loosen the enforcement around triple equals. Double equals really does do some strange and unintuitive things in Javascript and even if it's useful here it's harmful elsewhere.

Reply:
Okay. But on a second thought, I rather leave file unchanged in this PR. Change reverted.